Compile phase fails on the tock module

When i run the `mvn compile` command on the Tock project , i get the following error :

`[ERROR] Failed to execute goal org.apache.maven.plugins:maven-dependency-plugin:3.1.2:copy-dependencies (copy) on project tock-bot-admin-kotlin-compiler-core: Artifact has not been packaged yet. When used on reactor artifact, copy should be executed after packaging: see MDEP-187. -> [Help 1]`

The problem was discussed on your gitter with @francoisno :  

> François
> @francoisno
> 14:00
> Bonjour @FabienPennec en effet le plugin maven-dependency-plugin est utilisé pour copier le classpath dont dépend le compilateur dans un dossier temporaire (target/dependency), utile à l'exécution des tests unitaires du compilateur (voir KotlinCompilerTest#L34)
> 
> François
> @francoisno
> 14:02
> Malheureusement si on n'exécute pas un build jusqu'à la phase package au moins (donc install c'est ok), Maven au moment du build ne 'trouve pas' ces JARs pour les copier, puisqu'ils n'ont pas été construits
> 
> François
> @francoisno
> 14:07
> cela dit, la copie des dépendances pourrait se faire uniquement à la phase test et ton mvn compile se passerait bien, je vais regarder si on peut faire ça voire trouver une meilleure solution que copier les JARs dans un dossier
> 

A solution would be appreciated. :)

--- a/bot/admin/kotlin-compiler/core/pom.xml
+++ b/bot/admin/kotlin-compiler/core/pom.xml
@@ -68,22 +68,4 @@
         </dependency>
     </dependencies>
 
-    <build>
-        <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-dependency-plugin</artifactId>
-                <executions>
-                    <execution>
-                        <id>copy</id>
-                        <phase>validate</phase>
-                        <goals>
-                            <goal>copy-dependencies</goal>
-                        </goals>
-                    </execution>
-                </executions>
-            </plugin>
-        </plugins>
-    </build>
-
 </project>

--- a/bot/admin/kotlin-compiler/core/src/test/kotlin/KotlinCompilerTest.kt
+++ b/bot/admin/kotlin-compiler/core/src/test/kotlin/KotlinCompilerTest.kt
@@ -31,7 +31,7 @@ class KotlinCompilerTest {
         @BeforeAll
         @JvmStatic
         fun beforeClass() {
-            KotlinCompiler.init(listOf("target/dependency/", "target/test-classes/"))
+            KotlinCompiler.init(listOf("target/test-classes/"))
         }
 
         var mark = false

